apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/gqloperationdefinition.kt

```diff
@@ -12,4 +12,5 @@ fun GQLOperationDefinition.rootTypeDefinition(schema: Schema) = when (operationT
 fun GQLOperationDefinition.validate(
     schema: Schema,
     fragments: Map<String, GQLFragmentDefinition>,
-) = ExecutableValidationScope(schema, fragments).validateOperation(this)
\ No newline at end of file
+    fieldOnDisjointTypesMustMerge: Boolean,
```

Comment:
To avoid a breaking change, add a default value for `fieldsOnDisjointTypesMustMerge` and `@JvmOverloads`

Reply:
Then you can run `./gradle apiDump` to generate the compatibility report and push it (the CI checks it).

Reply:
Thanks for the suggestions. I'll update